src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/GovUk.Education.ExploreEducationStatistics.Content.ViewModels.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
We don't have to, but these settings mean we could remove any unnecessary `#nullable enable` and unneeded `System` imports from all the view models (which is basically all of them).

It's a little bit extra, but might be nice to do given we're here.

If you're not inclined to do it as part of this work, then would suggest we still keep these settings enabled anyway as they're sensible defaults for new subprojects.

Reply:
Good spot :) Sorted it as part of this work